packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -51,6 +51,16 @@ export interface ISemanticColors {
 
   //// Base slots
 
+  /**
```

Comment:
We wanted to add a bodyStandoutBackground too right?

Reply:
Per the designer: "for standouts, we have bodyFrameBackground (hub nav in header) and standoutCardBackground (news hub/top news, some cards)" - not bodyStandoutBackground.

The designer would prefer to add new semantic slots as each redline is finalized. The two slots mentioned above (bodyFrameBackground and standoutCardBackground) are not applicable to the events webpart, so they are slated to be added later.